src/sentry/snuba/events.py

```diff
@@ -480,6 +480,14 @@ class Columns(Enum):
         discover_name="exception_stacks.mechanism_handled",
         alias="error.handled",
     )
+    ERROR_MAIN_THREAD = Column(
+        group_name="events.exception_main_thread",
+        event_name="exception_main_thread",
+        transaction_name=None,
+        discover_name="exception_main_thread",
+        issue_platform_name="exception_main_thread",
```

Comment:
I don't know if this is needed, just copied over the `issue_platform_name`

Reply:
@wmak do you know about this?

Reply:
looks like @wedamija introduced it here: https://github.com/getsentry/sentry/pull/42279

Reply:
Is this a new column that was added to the errors and discover dataset? Or does it already exist in snuba and we're just exposing it?

Reply:
This is a new column, details [here](https://github.com/getsentry/snuba/pull/3863).

Reply:
I would suggest leaving it not set so it defaults to `None`. Our dataset shouldn't support that column.

Reply:
Could you also add the new search term to this list: https://github.com/getsentry/sentry/blob/c03119c02e0b853347de672fefcfd78b04579b24/src/sentry/issues/search.py#L259

and update this test:
https://github.com/getsentry/sentry/blob/cc6f7a2a61d77ccb386d5221ba86944f6737d7d1/tests/snuba/search/test_backend.py#L2494

Reply:
Probably worthwhile to add some positive and negative tests to `sentry/tests/snuba/search/test_backend.py` which does the actual search.

Reply:
Also to add some tests to the discover events endpoint: https://github.com/getsentry/sentry/blob/29080a2eb096d7c63d359f48ffb1ce82963701cf/tests/snuba/api/endpoints/test_organization_events.py#L1046